src/math/util.hpp

```diff
@@ -17,6 +17,8 @@
 #ifndef HEADER_SUPERTUX_MATH_UTIL_HPP
 #define HEADER_SUPERTUX_MATH_UTIL_HPP
 
+#include <math.h>
```

Comment:
Is this include needed?

Reply:
It appears so, despite that I didn't change anything else...